charts/node-exporter/values.yaml

```diff
@@ -0,0 +1,15 @@
+prometheus-node-exporter:
```

Comment:
@sjmiller609 I think we'll want to maintain our ability to pin our stuff to certain node pools. Can we somehow pass some node pool settings down through to this chart, based on our `global` settings?

Reply:
Ah right. Thanks for the reminder greg. I will need to ensure that tolerations and affinity can be passed. We will make use of the toleration configuration for cloud.

Reply:
After considering this, I believe that no change is required. We can pass down tolerations from the top-level chart like this:
```
node-exporter:
  prometheus-node-exporter:
    tolerations: []
```
We do not want to use the global configuration "platformNodePool" because this should be applied to all nodes - we want to just pass in tolerations for all taints, and allow the daemonset to run on all nodes.

Reply:
I'm just thinking of the enterprise use-case where we may or may not have control of an entire cluster, maybe a specific namespace and node pool. If we couldn't deploy the daemonset to all nodes, would it still be useful to have on the nodes that our stuff is running?

Reply:
If there are enterprise use cases where we are limited to what nodes we can be installed on then yes we should have the ability to add node selectors or modify the tolerations.

Reply:
we do have that capability (see my most recent comment in this thread)